rotkehlchen/api/v1/schemas.py

```diff
@@ -2575,6 +2575,40 @@ def transform_data(
             blockchain=data['blockchain'],
         )
 
+    @validates_schema
+    def validate_schema(
+            self,
+            data: dict[str, Any],
+            **_kwargs: Any,
+    ) -> None:
+        if data['blockchain'] is None:
+            return
+
+        blockchain = cast(SupportedBlockchain, data['blockchain'])
+        supported_blockchain_type = blockchain.get_chain_type()
+        if supported_blockchain_type == 'evm':
+            try:
+                to_checksum_address(data['address'])
```

Comment:
just use deserialize_evm_address here. Why TypeError? Our `deserialize_evm_address` does the same thing, is used across the entire codebase and never catches that.